auth_saml/__manifest__.py

```diff
@@ -13,7 +14,7 @@
         'web',
     ],
     "external_dependencies": {
-        "python": ["lasso"],
+        "python": ["saml2"],
     },
     "demo": [
         'demo/auth_saml.xml',
```

Comment:
demo file  show the following warning:

`2020-12-18 04:53:45,018 12075 WARNING auth odoo.models: auth.saml.provider.create() with unknown fields: sp_metadata, sp_pkey`